<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>172</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CFDICalc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/musashi_mochi.png</normaloff>:/musashi_mochi.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Subtotal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalDisplay">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="subtotalDisplay">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="copyTotal">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Copiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copySubtotal">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Copiar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>299</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archi&amp;vo</string>
    </property>
    <addaction name="actionAbrir_CFDI"/>
    <addaction name="actionAcerca_de"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuFiltrar">
    <property name="title">
     <string>Filtrar</string>
    </property>
    <addaction name="actionTodos"/>
    <addaction name="separator"/>
    <addaction name="actionG01"/>
    <addaction name="actionG02"/>
    <addaction name="actionG03"/>
    <addaction name="separator"/>
    <addaction name="actionI01"/>
    <addaction name="actionI02"/>
    <addaction name="actionI03"/>
    <addaction name="actionI04"/>
    <addaction name="actionI05"/>
    <addaction name="actionI06"/>
    <addaction name="actionI07"/>
    <addaction name="actionI08"/>
    <addaction name="separator"/>
    <addaction name="actionD01"/>
    <addaction name="actionD02"/>
    <addaction name="actionD03"/>
    <addaction name="actionD04"/>
    <addaction name="actionD05"/>
    <addaction name="actionD06"/>
    <addaction name="actionD07"/>
    <addaction name="actionD08"/>
    <addaction name="actionD09"/>
    <addaction name="actionD10"/>
    <addaction name="separator"/>
    <addaction name="actionS01"/>
    <addaction name="separator"/>
    <addaction name="actionCP01"/>
    <addaction name="actionCN01"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuFiltrar"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir_CFDI">
   <property name="text">
    <string>&amp;Abrir CFDI...</string>
   </property>
  </action>
  <action name="actionTodos">
   <property name="text">
    <string>&amp;Todos</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>TOTAL</string>
   </property>
  </action>
  <action name="actionG01">
   <property name="text">
    <string>&amp;G01 Adquisición de mercancías</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>G01</string>
   </property>
  </action>
  <action name="actionG02">
   <property name="text">
    <string>G02 Devoluciones, descuentos o &amp;bonificaciones</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>G02</string>
   </property>
  </action>
  <action name="actionG03">
   <property name="text">
    <string>G&amp;03 Gastos en general</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>G03</string>
   </property>
  </action>
  <action name="actionI01">
   <property name="text">
    <string>&amp;I01 Construcciones</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>I01</string>
   </property>
  </action>
  <action name="actionI02">
   <property name="text">
    <string>I02 &amp;Mobiliario y equipo de oficina por inversiones</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>I02</string>
   </property>
  </action>
  <action name="actionI03">
   <property name="text">
    <string>I03 &amp;Equipo de transporte</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>I03</string>
   </property>
  </action>
  <action name="actionI04">
   <property name="text">
    <string>I04 Equipo de cómputo &amp;y accesorios</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>I04</string>
   </property>
  </action>
  <action name="actionI05">
   <property name="text">
    <string>I0&amp;5 Dados, troqueles, moldes, matrices y herramental</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>I05</string>
   </property>
  </action>
  <action name="actionI06">
   <property name="text">
    <string>I0&amp;6 Comunicaciones telefónicas</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>I06</string>
   </property>
  </action>
  <action name="actionI07">
   <property name="text">
    <string>I0&amp;7 Comunicaciones satelitales</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>I07</string>
   </property>
  </action>
  <action name="actionI08">
   <property name="text">
    <string>I08 &amp;Otra maquinaria y equipo</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>I08</string>
   </property>
  </action>
  <action name="actionD01">
   <property name="text">
    <string>&amp;D01 Honorarios médicos, dentales y gastos hospitalarios</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>D01</string>
   </property>
  </action>
  <action name="actionD02">
   <property name="text">
    <string>D0&amp;2 Gastos médicos por incapacidad o discapacidad</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>D02</string>
   </property>
  </action>
  <action name="actionD03">
   <property name="text">
    <string>D03 Gastos &amp;funerales</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>D03</string>
   </property>
  </action>
  <action name="actionD04">
   <property name="text">
    <string>D0&amp;4 Donativos</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>D04</string>
   </property>
  </action>
  <action name="actionD05">
   <property name="text">
    <string>D05 Intereses &amp;reales efectivamente pagados por créditos hipotecarios (casa habitación)</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>D05</string>
   </property>
  </action>
  <action name="actionD06">
   <property name="text">
    <string>D06 Aportaciones &amp;voluntarias al SAR</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>D06</string>
   </property>
  </action>
  <action name="actionD07">
   <property name="text">
    <string>D07 &amp;Primas por seguros de gastos médicos</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>D07</string>
   </property>
  </action>
  <action name="actionD08">
   <property name="text">
    <string>D0&amp;8 Gastos de transportación escolar obligatorio</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>D08</string>
   </property>
  </action>
  <action name="actionD09">
   <property name="text">
    <string>D09 Depósitos en cuentas para el ahorro, primas &amp;que tengan como base planes de pensiones</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>D09</string>
   </property>
  </action>
  <action name="actionD10">
   <property name="text">
    <string>D&amp;10 Pagos por servicios educativos (colegiaturas)</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>D10</string>
   </property>
  </action>
  <action name="actionS01">
   <property name="text">
    <string>&amp;S01 Sin efectos fiscales</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>S01</string>
   </property>
  </action>
  <action name="actionCP01">
   <property name="text">
    <string>&amp;CP01 Pagos</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>CP01</string>
   </property>
  </action>
  <action name="actionCN01">
   <property name="text">
    <string>CN01 &amp;Nómina</string>
   </property>
   <property name="UsoCFDI" stdset="0">
    <string>CN01</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca &amp;de...</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
